beets/ui/commands.py

```diff
@@ -39,6 +42,7 @@
 from beets.util.confit import _package_path
 
 VARIOUS_ARTISTS = u'Various Artists'
+ExtraChoice = namedtuple('ExtraChoice', ['plugin', 'id', 'long', 'callback'])
```

Comment:
Great! I like namedtuples; they really can improve readability.

What do you think about calling this something more generic, like `PromptChoice`, `UIChoice`, or just plain `Choice`? That way, we could (someday) use the same infrastructure for built-in choices as plugins use for "extra" choices. As much as is reasonable, it's nice to use common tooling for both plugin and non-plugin functionality so we can avoid duplication.